content/en/logs/guide/getting-started-lwl.md

```diff
@@ -0,0 +1,132 @@
+5. Select **Add an Exclusion Filter**.
+6. Input the filter name, define the exclusion query by pasting the search query you copied, and set an exclusion percentage.
+
+{{< img src="logs/guide/getting-started-lwl/exclusion_filter.gif" alt="Exclusion Filter" responsive="true" style="width:100%;">}}
+
+**Note**: If a log matches several exclusion filters, only the first exclusion filter rule is applied. A log is not sampled or excluded multiple times by different exclusion filters.
+
+In this example, the service status `info` pattern `Updating recommendations with customer_id=* & url=shops/*/*` is filtered with an exclusion filter. Removing any high volume logging pattern similar to this one from Log Explorer will help you drill down and identify issues quicker. However, these logs are **only** removed from the Log Explorer view. They are still ingested, indexed, and available to view in [Live Tail][6], sent to [log archives][7], or used to [generate metrics][8].
+
+{{< img src="logs/guide/getting-started-lwl/live_tail.gif" alt="Live Tail" responsive="true" style="width:100%;">}}
+
+Exclusion filters can be disabled at any time by toggling the disable option to the right of the filter. They can also be modified and removed by hovering over the filter and selecting the edit or delete option.
+
+## 4. Generate metrics for excluded logs
+
+Once a log pattern is excluded from Log Explorer, you can still track KPIs over time at the ingest level by creating a new [custom log-based metric][9]. 
+
+### Add a new log-based metric
+
+**To generate a new log-based metric based on your log pattern**:
+
+1. In your Datadog account, hover over **Logs** in the main menu, select **Generate Metrics**, and then click the **New Metric+** button in the top right corner.
+2. Under **Define Query**, input the search query you copied and pasted into the pattern exclusion filter. (e.g., as per the example above: `service:web-store status:info "updating recommendations with customer_id" "url shops"`)
+3. Select the field you would like to track: Select `*` to generate a count of all logs matching your query or enter a measure (e.g., `@duration`) to aggregate a numeric value and create its corresponding count, min, max, sum, and avg aggregated metrics.
+4. Add dimensions to group: Select log attributes or tag keys to apply to the generated log-based metric to transform them into tags following the `<KEY>:<VALUE>` format. Log-based metrics are considered custom metrics. Avoid grouping by unbounded or extremely high cardinality attributes like timestamps, user IDs, request IDs, or session IDs to avert impacting your billing.
+5. Name your metric: Log-based metric names must follow the naming metric convention.
+
+{{< img src="logs/guide/getting-started-lwl/custom_metric.gif" alt="Generate Metric" responsive="true" style="width:100%;">}}
```

Comment:
Gif here is broken

Reply:
Just viewed locally and it is working 🤷‍♀hm...